source/includes/administration/_organization_discounts.md

```diff
@@ -0,0 +1,291 @@
+## Organization Discounts
+
+The discount allows the assignment of a percentage discount or credit to an organization.
+
+<!-------------------- LIST ORGANIZATION DISCOUNTS -------------------->
+### List organization discounts
+
+`GET /organizations/:organization_id/discounts?type=:type`
+
+Retrieve the list of discounts associated with an organization.
+
+```shell
+# Retrieve organization discount list
+curl "https://cloudmc_endpoint/rest/organizations/efd32752-c6f2-45cf-b494-cc6be8a45845/discounts?type=PERCENTAGE" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "applyToNewCustomersOnly": true,
+      "discountedProducts": {},
+      "durationDays": 60,
+      "type": "PERCENTAGE",
+      "packageDiscount": 20.0,
+      "scope" "ALL_PRODUCTS",
+      "discountedCategories": {},
+      "name": {
+        "en": "Summer Discount",
+        "fr": "Réduction Estival"
+      },
+      "id": "18db7bc6-8be1-48bb-bab1-77a7d696fa3b",
+      "startDate": "2021-07-20T15:57:16.132Z",
+      "status": "CURRENT"
+    }
+  ]
+}
+```
+
+Optional Query Parameters | &nbsp;
+---------- | -----------
+`type`<br/>*enum* | The type of the discount. It could be either "PERCENTAGE" or "CREDIT".
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*string* | The string of the discount.
+`discountedProducts`<br/>*object[string, number]* | A mapping of the desired priced product IDs and discounts. All pricing products specified will have the discount value applied to them.
+`durationDays`<br/>*integer* | Duration of the discount once it has been applied to a customer. If not provided the discount will last indefinitely, or until credit values are reached.
+`type`<br/>*enum* | The type of the discount. It could be either "PERCENTAGE" or "CREDIT".
+`packageDiscount`<br/>*number* | The discount value that will be applied to all products within the applied pricing.
+`scope`<br/>*enum* | The scope of the discount. It could be either "ALL_PRODUCTS", "CATEGORIES" or "PRODUCTS".
+`discountedCategories`<br/>*object[string, number]* | A mapping between category IDs and discount values. All pricing products within a category will have the discount value applied to them.
```

Comment:
Can you standardize the description of this field with the description of `discountedProducts`. Also adding an example response body would be nice